test/performance/odp_l2fwd.c

```diff
@@ -791,6 +792,18 @@ static int create_pktio(const char *dev, int idx, int num_rx, int num_tx,
 
 	odp_pktio_config(pktio, &config);
 
+	if (pktio_capa.set_op.op.promisc_mode) {
```

Comment:
Simply fail if user has enabled promisc mode but the implementation doesn't support it.

Reply:
ok.